### ISSUE TITLE
Support/RFC: Use the keypad as single-handed "browsing remote"

### ISSUE THREAD

**Issue by pataquets** [tridactyl/tridactyl#4845]
## Making a feature request  / getting help
-   Tridactyl version (`:version`): 1.23.0
-   Firefox version (Top right menu > Help > About Firefox): 120.0

## Use the keypad as single-handed "browsing remote"
I was thinking on having a single-handed navigation using just the numeric keypad block. It has some advantages, especially for long browsing sessions:
- The key layout is ortholinear (i.e. key rows are stacked instaead of staggered), which migh be ease some RSI.
- Numpad is a rightmost on keyboard, so less arm twisting towards the keyboard center.
- There are plenty of add-on keypads, even wireless, which might be of used as a sort of "browsing remote", ideal for [10-foot user interfaces](https://en.wikipedia.org/wiki/10-foot_user_interface).

As Tridactyl can provide numeric hints, being able to bind commands to keys in the numpad would be perfect. As I start reading [the docs](https://tridactyl.xyz/build/static/docs/modules/_src_excmds_.html#bind), I get to [Mozilla's docs on keyboard events](https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values#numeric_keypad_keys). However, binds like `Multiply` doesn't seem to work (as opposed to `*`). I'd like to all add,substract and number keys to be differentiated from main keyboard counterparts, to assign the different funcitions.

Is that even possible? What would the syntax be?

**Comment by bovine3dom** [comment]
Hi, we don't support directly binding to `key` objects, but you can use the keyboardlayout settings to override each `keyboardEvent.code` from https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_code_values - which, NB, is not the same as the page you were on - you want to bind to, to a character that you are unlikely to type (such as an emoji from https://emoji.muan.co/ )

```
set keyboardlayoutbase qwerty # whatever your keyboard is
set keyboardlayoutforce true
set keyboardlayoutoverrides.NumpadMultiply ✖️✖️ # second character is for shift+key, make it different if you like. sometimes it will work
bind ✖️ scrollline 10
reload
```

Code is untested because I don't have any keyboards that have numpads :)

**Comment by pataquets** [comment]
At some point, my addon went updated, so I decided to try it:
``` console
:version
1.24.1
```
As per your instructions, I did:
``` js
set keyboardlayoutforce true
set keyboardlayoutoverrides.Numpad5 f
```
Although it worked, I've found it comes with a major drawback: my keyboard layout changed from Spanish to US :fearful:, which makes it hardly usable :cry:.
Luckily, I've found how to deal with this in [keyboardlayoutforce docs](https://github.com/tridactyl/tridactyl/blob/369783e87f7da86e3f8c10b3791b4bf592719f00/src/lib/config.ts#L1381) (dropping here for reference). I'll get around to it when I have some more time.

**Comment by bovine3dom** [comment]
Yeah, just follow the instructions https://tridactyl.xyz/build/static/docs/classes/_src_lib_config_.default_config.html#keyboardlayoutbase and I'd be happy to add a Spanish layout
